### ISSUE TITLE
Name?

### ISSUE THREAD

**Issue by jakirkham** [loriab/anvil#1]
Was called `anvil` previously. Maybe that should stay? Have prefixed it with `conda-forge-` so it is easier to find. Though am open to suggestions. Please share your thoughts.

**Comment by pelson** [comment]
Names: The single hardest thing in software development... 😄 

So anvil was nice because it is the most obvious part of a forge, but I agree that the linux-anvil is a good name because we hammer so many recipes against it. In many ways, metaphorically, this repo should reflect the finished product from the forge. From Wikipedia for forging: "the traditional products were kitchenware, hardware, hand tools, edged weapons, and jewellery".

We could definitely open the question to a wider audience, either on gitter or twitter if we wanted.

**Comment by jakirkham** [comment]
Yeah, I thought about mentioning that anvil felt like a development tool as opposed to a finish product. Though I could see the counterargument as we do plan on using it for development too. Sure maybe we can gather up a few suggestions from people and then do a survey/vote.

**Comment by richardotis** [comment]
`conda-forge-bundle`

**Comment by jakirkham** [comment]
Reaching out to @conda-forge/core generally. Any thoughts on the name for the installer?

**Comment by ngoldbaum** [comment]
In analogy to the acaconda and miniconda distributions that Continuum already distributes, you could call it something like libreconda or communityconda. That might make it more obvious to your the audience that you are getting a conda-based software distrubition. I'm not sure if a name like that would step on any of Continuum's trademarks...

**Comment by minrk** [comment]
anvil makes sense in the whole CI conda-forge metaphor, so if what it's *for* is the baseline for conda-forge CI makes sense. But for telling a public user to get started using conda, what I will want to tell people is "Get conda from conda-forge" or "Get the conda-forge conda installer".

Given that we already deal with confusion re: miniconda vs anaconda, I'd like to keep it as simple as possible, e.g. something along the lines of `conda-installer` or `conda-forge-install`.

**Comment by jakirkham** [comment]
These are all good ideas. Thanks for sharing them.

One of the big motivators to put work into this was installing it on CIs instead of Miniconda to speed up our builds (i.e. starting with the desired `root` environment instead of upgrading to it). However, the secondary motivator was to make it easy for others to install the conda-forge stack in a community friendly way. 😄

As such I agree we should move away from having anvil in the name. Not only because it is unclear what it means, but it implies there is something unfinished about it like `conda-forge/linux-anvil` (our Linux development Docker container). The name should be something that indicates a finished product ready for outside consumption.

**Comment by patricksnape** [comment]
There's a part of me that likes keeping the name in the vein of the 'smithy' nomenclature - but I think I'm actually leaning towards the clarity of something boring like `conda-forge-install` which feels very transparent. I do like the word 'foundry' however...

**Comment by ngoldbaum** [comment]
How about some more names that aren't as obvious: `conda-billet` or `conda-ingot`, to indicate something of a finished product or something to directly build on.

**Comment by kain88-de** [comment]
> I think I'm actually leaning towards the clarity of something boring like `conda-forge-install` which feels very transparent

Yes `conda-forge-installer` is the most transparent and user friendly. The meaning is clear to everybody, even people who don't speak english that well. The other names are nice but will be more confusing for non-native speaker that don't know what a anvil/ingot/... is.
